bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Widget.java

```diff
@@ -1137,7 +1137,8 @@ void reskinWidget() {
 boolean sendDragEvent (int button, int x, int y) {
 	Event event = new Event ();
 	event.button = button;
-	event.setLocationInPixels(x, y); // In Pixels
+	int zoom = getZoom();
+	event.setLocation(DPIUtil.scaleDown(x, zoom), DPIUtil.scaleDown(y, zoom)); // In Pixels
```

Comment:
The comment is now outdated, right?
```suggestion
	event.setLocation(DPIUtil.scaleDown(x, zoom), DPIUtil.scaleDown(y, zoom));
```